src/main/java/tutu/task/ToDo.java

```diff
@@ -0,0 +1,27 @@
+package tutu.task;
+
+import tutu.exception.InvalidInputException;
+
+/** Represents a ToDo task. */
+public class ToDo extends Task {
+    /**
+     * Constructor to create a ToDo object
+     * @param task Command input from user.
+     * @throws InvalidInputException If input is invalid.
+     */
+    public ToDo(String task) throws InvalidInputException {
+        super(task);
+        if (super.task.length() < 6) {
```

Comment:
I feel like using 6 as a magic number could be avoided. Perhaps you can consider introducing a constant variable like `private final int MINIMUM_DESCRIPTION_LENGTH = 6`?